doc/man5/flux-config-queues.rst

```diff
@@ -0,0 +1,88 @@
+=====================
+flux-config-queues(5)
+=====================
+
+
+DESCRIPTION
+===========
+
+The ``queues`` table configures job queues, as described in RFC 33.
+Normally, Flux has a single anonymous queue, but when queues are configured,
+all queues are named, and a job submitted without a queue name is rejected
+unless a default queue is configured.
+
+Each key in the ``queues`` table is a queue name, whose value is a table
+consisting of the following optional keys:
+
+requires (array)
+   A list of resource property names that selects a subset of the total
+   configured resources for the queue.  Each property name is a string, as
+   defined in RFC 20.  If multiple properties are listed, they are combined
+   in a logical *and* operation.  Properties are attached to resources in the
+   ``resource`` table as described in :man5:`flux-config-resource`.
+
+policy (table)
+   A policy table as described in :man5:`flux-config-policy` that overrides
+   the general system policy for jobs submitted to this queue.
+
+A default queue name may be configured by setting
+``policy.jobspec.defaults.system.queue`` as described in
+:man5:`flux-config-policy`.
+
+
+EXAMPLE
+=======
+
+::
+
+   [ingest]
+   validator.plugins = ["feasibility", "jobspec"]
+   frobnicator.plugins = ["defaults", "constraints"]
+
+   [[resource.config]]
+   hosts = test[0-7]
+   properties = ["debug"]
+
+   [[resource.config]]
+   hosts = test[8-127]
+   properties = ["batch"]
+
+   [queues.debug]
+   policy.limits.duration = "30m"
+   requires = [ "debug" ]
+
+   [queues.batch]
+   policy.limits.duration = "8h"
+   requires = [ "batch" ]
+
+   [policy.jobspec.defaults.system]
+   queue = "batch"
+
+   [sched-fluxion-qmanager]
+   queues = "batch debug"
+   queue-policy-per-queue="batch:easy debug:fcfs"
+
+   [sched-fluxion-resource]
+   match-policy = "lonodex"
+   match-format = "rv1_nosched"
+
+
+CAVEATS
+=======
+
+Queue resources should not overlap.
```

Comment:
Oh, I was going to ask about this caveat. Why can't queue resources overlap (is this a Fluxion constraint?).

Reply:
Fluxion allows it, but some issues were identified in flux-framework/flux-sched#939.

Some use cases might work with Fluxion as is, like opening up a *stopped* queue to select users before a DAT, then when the DAT comes around, stop/drain the normal queues and start the DAT queue.   That particular one would require the job manager to support multiple queues though. 

Maybe there are other cases, but I thought it best to not advertise it at this point since the near term goal is partitioned resources.

Reply:
Ah, thanks, I had forgotten about that. I do know we were talking about allowing a queue with access to all resources (and we do have a `pall` partition on many clusters), and this caveat prevents that option. I agree we can ignore it for now as long as the concept of non-overlapping resource sets isn't baked into the design.